code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -150,7 +150,8 @@ GLOBAL_LIST_INIT(wood_recipes, list ( \
 
 	new/datum/stack_recipe("wooden door", /obj/structure/mineral_door/wood, 10, time = 2 SECONDS, max_per_turf = STACK_RECIPE_ONE_PER_TILE, on_floor = TRUE), \
 	new/datum/stack_recipe("coffin", /obj/structure/closet/coffin, 5, time = 1.5 SECONDS, max_per_turf = STACK_RECIPE_ONE_PER_TILE, on_floor = TRUE), \
-	new/datum/stack_recipe("baseball bat", /obj/item/weapon/baseballbat, 10, time = 2 SECONDS, on_floor = TRUE) \
+	new/datum/stack_recipe("baseball bat", /obj/item/weapon/baseballbat, 10, time = 2 SECONDS, on_floor = TRUE), \
+	new/datum/stack_recipe("wooden crate", /obj/structure/largecrate/packed, 1, time = 2 SECONDS, on_floor = TRUE) \
```

Comment:
I feel like 1 wood isn't very much and maybe 5 would be more sensible, but I also have no sense for price or availability.

Reply:
I did 1 because deconning also gives 1 wood, I can increase it on both counts though